README.md

```diff
@@ -89,6 +89,21 @@ Once you have either downloaded the VSIX file or created it you can now install
 
 Please refer to the Wiki page at https://github.com/apache/daffodil-vscode/wiki
 
+To debug a daffodil schema, choose open a folder from the File menus, then select a dfdl.xsd file. Once the file loads in VS Code, then set a desired breakpoint.
+![Setting Breakpoints](./images/SetBreakpoint.gif)
+
+To start debugging, click on the debug button on the side panel, then click the RUN AND DEBUG button with the dropdown set to Ask for file name.
+![Debugging](./images/StartDebug.gif)
+
+To step through the schema code, use the step over button
+![Step Over](./images/StepThrough.gif)
+
+To view the infoset, use the veiw the XML infoset button
+![View Infoset](./images/ViewInfoset.gif)
+
+To view the data file in hex, use the view hex button
+![View Hex](./images/ViewHex.gif)
+
```

Comment:
These images are very wide, like 1800 pixels. When they are rendered in a github readme, e.g.

https://github.com/apache/daffodil-vscode/blob/f838c5c753e97aeed30c71db76d56396e34e40b3/README.md

They end up getting scaled down to fit the width and so we lose a lot of quality--images get blurry and bit hard to read. It looks like anything over 1012 pixels is going to get scaled. I'd suggest creating these with a smaller VS Code window so they fit the full width of a github page without needing to be scaled.

Reply:
I captured the images on a laptop, so the images looked small to me.  My resolution on the laptop screen is 1920 x 1080.  Should I create the images at a lower resolution 800 x 600?

Reply:
I'll remove the debugger use info from the original readme.  But I'll wait on your feedback to the other comments before proceeding with anything else.